File: python/day-01/part-2.py
with open("input", "r") as file:
    input = file.read()

    floor = 0
    position = 1

    for char in input:
        if char == "(":
            floor = floor + 1
        elif char == ")":
            floor = floor - 1

        if floor == -1:
            break

        position += 1
    
    print(position)

File: javascript-web/day-01/part-2.js
// console.log(input);

let floor = 0;
let position = 1;

for (const char of input) {
  if (char === "(") {
    floor++;
  } else if (char === ")") {
    floor--;
  }

  if (floor === -1) break;

  position++;
}

console.log("Part 2:", position);
